src/components/ChecklistItem.tsx

```diff
@@ -0,0 +1,241 @@
+import { useKeyboard } from '@react-aria/interactions'
+import {
+  ComponentPropsWithRef, Dispatch, ReactElement, useEffect, useRef, useState,
+} from 'react'
+import AnimateHeight from 'react-animate-height'
+import styled from 'styled-components'
+
+import CaretDownIcon from './icons/CaretDownIcon'
+import SuccessIcon from './icons/SuccessIcon'
+
+const heightAnimationDuration = 666 // 666ms
```

Comment:
Interesting choice of number. 🤔
Though I do think the animation feels a little slow. I'd consider speeding this up to something more like 200-500. Just a personal opinion though.

Reply:
I have decreased some animation times to make them faster. Indeed it looks smoother.